modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectStops.java

```diff
@@ -370,4 +378,41 @@ JSONObject reduceJsonForRange(JSONObject jsonObject, long[] range) {
         }
         return layerZoomLevels;
     }
+
+    public class LayerStops {
+        public boolean propertyStops = false;
+        public boolean zoomStops = false;
+        public boolean zoomPropertyStops = false;
+        public boolean hasStops = false;
+
+        public LayerStops() {
+            if (zoomPropertyStops || zoomStops || propertyStops) {
+                this.hasStops = true;
+            }
```

Comment:
Since all three of `zoomPropertyStops || zoomStops || propertyStops` are initialized to false, this code will never get executed; you can't set them any other way before calling this constructor.

Consequently, there is no reason to even define a constructor, the default empty constructor is functionally identical.